solr_plugins/Bibelstellensuche/tests/de/uni_tuebingen/ub/ixTheo/bibleRangeSearch/BibleRangeTests.java

```diff
@@ -0,0 +1,127 @@
+package de.uni_tuebingen.ub.ixTheo.bibleRangeSearch;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class BibleRangeTests {
+
+	@Test
+	public void testGetDistance() {
+		Range r1, r2;
+
+		r1 = new Range(0, 2);
+		r2 = new Range(1, 2);
+		assertEquals(1, r1.getDistance(r2));
+
+		r1 = new Range(0, 3);
+		r2 = new Range(1, 2);
+		assertEquals(2, r1.getDistance(r2));
+
+		r1 = new Range(0, 2);
+		r2 = new Range(2, 4);
+		assertEquals(4, r1.getDistance(r2));
+
+		r1 = new Range(0, 2);
+		r2 = new Range(5, 7);
+		assertEquals(Range.OUT_OF_RANGE, r1.getDistance(r2));
+	}
+
+	@Test
+	public void testGetMinimumDistance() {
+		Range r1;
```

Comment:
Combine this and the following line, please.  In general, always try to combine declaration and initialization.